rotkehlchen/data_import/importers/nexo.py

```diff
@@ -80,16 +84,20 @@ def _consume_nexo(
         else:
             raise SkippedCSVEntry('Ignoring rejected entry.')
 
-        asset = asset_from_nexo(csv_row['Output Currency'])
-        amount = deserialize_asset_amount_force_positive(csv_row['Output Amount'])
         entry_type = csv_row['Type']
         transaction = csv_row['Transaction']
+        amount = deserialize_asset_amount_force_positive(csv_row['Output Amount'])
+        try:
+            asset = asset_from_nexo(csv_row['Output Currency'])
+        except UnknownAsset:
+            # from the csv a user shared, some entries with type of
+            # interest and referral bonus did not contain an output currency
```

Comment:
if they did not contain an output currency isn't iut best to do csv_row.get('Output Currency') and compare with None? Instead of calling the entire function?

Reply:
ah yes, thanks.